samples/csharp_dotnetcore/05.multi-turn-prompt/readme.md

```diff
@@ -31,9 +31,7 @@ developers to test and debug their bots on localhost or running remotely through
 - File -> Open bot and navigate to `botbuilder-samples/samples/csharp_dotnetcore/05.multi-turn-prompt` folder.
 - Select `BotConfiguration.bot` file.
 # Deploy this bot to Azure
-You can use the [MSBot](https://github.com/microsoft/botbuilder-tools) Bot Builder CLI tool to clone and configure any services this sample depends on. 
-
-To install all Bot Builder tools - 

```

Comment:
This line shouldn't be combined with the one above.

Reply:
The line has been fixed with the correct text